ocaml/tests/test_xenopsd_metadata.ml

```diff
@@ -226,18 +194,14 @@ module VideoRam = Generic.Make(Generic.EncapsulateState(struct
                                    {oc=[]; platform=[]}, 4;
                                    (* Specifying a different amount of videoram works. *)
                                    {oc=[]; platform=["videoram", "8"]}, 8;
-                                   (* Default videoram should be 16MiB for vGPU. *)
-                                   {oc = []; platform=vgpu_platform_data}, 16;
-                                   (* Insufficient videoram values should be overridden for vGPU. *)
-                                   {oc = []; platform=vgpu_platform_data @ ["videoram", "8"]}, 16;
-                                   (* videoram values larger than the default should be allowed for vGPU. *)
-                                   {oc = []; platform=vgpu_platform_data @ ["videoram", "32"]}, 32;
                                    (* Other VGA options shouldn't affect the videoram setting. *)
                                    {oc = []; platform=["vga", "cirrus"]}, 4;
                                    {oc = []; platform=["vga", "cirrus"; "videoram", "8"]}, 8;
                                  ]
                                end))
 
+let test_uuid_base = "uuiduuid-uuid-uuid-uuid-uuiduuid"
```

Comment:
This is obviously not a uuid. It might not matter but I would prefer something that has at least the correct syntactical structure while still not being random.

```
let uuid_with_suffix suffix =
 Printf.sprintf "11111111-1111-1111-1111-11111111%s" suffix
```

Reply:
Agree. But I prefix zero's.